src/osdc/Objecter.cc

```diff
@@ -2680,6 +2684,7 @@ int Objecter::_calc_target(op_target_t *t, epoch_t *last_force_resend,
 			       &acting, &acting_primary);
   bool sort_bitwise = osdmap->test_flag(CEPH_OSDMAP_SORTBITWISE);
   unsigned prev_seed = ceph_stable_mod(pgid.ps(), t->pg_num, t->pg_num_mask);
+  // XXX: I think any_change means this is being called for second time.
```

Comment:
yeah, i think so. it could be some lingering ops that were sent to the osd before. or homeless ops, and we are  trying to find it a home with a new osdmap.